backend/lm_backend/api/schemas/feature.py

```diff
@@ -31,6 +31,16 @@ class FeatureUpdateSchema(BaseUpdateSchema):
     used: Optional[int] = None
 
 
+class FeatureUpdateByNameSchema(BaseUpdateSchema):
+    """
+    Represents the features in a feature configuration.
```

Comment:
I think this docstring isn't right.

Reply:
Is `"Represents the feature usage data that will be updated using the name as a filter."` better?

Reply:
Yes, thank you.